src/site/paragraphs/collapsible_panel.drupal.liquid

```diff
@@ -30,7 +30,8 @@
     <ul class="usa-unstyled-list">
         {% for accordionItem in entity.fieldVaParagraphs %}
             {% assign item = accordionItem.entity %}
-            <li id="{{ item.entityId | formatSharableLinkID: item.fieldTitle }}"
+            {% assign id= item.entityId | formatSharableLinkID: item.fieldTitle %}
+            <li id={{id}}
```

Comment:
I think you need quotes around `{{ id }}` right?

Reply:
hmm, i don't think so! it's working somehow. but i'm not sure

Reply:
Weird... I wonder if it's a string it's inlining the quotes then? Liquid is funky lol